examples/create-react-app-with-styled-components-engine/src/App.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Slider from '@material-ui/lab/SliderStyled';
+import { Typography } from '@material-ui/core';
```

Comment:
To match what we do in the demos?

```suggestion
import Typography from '@material-ui/core/Typography';
```